projects/plugins/mu-wpcom-plugin/composer.json

```diff
@@ -4,7 +4,8 @@
 	"type": "wordpress-plugin",
 	"license": "GPL-2.0-or-later",
 	"require": {
-		"automattic/jetpack-mu-wpcom": "@dev"
+		"automattic/jetpack-mu-wpcom": "@dev",
+		"automattic/scheduled-updates": "@dev"
```

Comment:
This should not be needed, the addition in `projects/packages/jetpack-mu-wpcom/composer.json` should already be bringing it in.

You can test that easily enough by reverting all the changes in `projects/plugins/mu-wpcom-plugin/`, then run `tools/fixup-project-versions.sh` and see whether it updates composer.lock with the same changes (except for the hash on line 7) as shown below.

Reply:
With the changes to `mu-wpcom-plugin` reverted, I'm unable to follow the WoA testing steps in PCYsg-Osp-p2/#woa-testing. The Scheduled Updates package doesn't end up in `vendors`. Am I missing a step there?

Reply:
Did you run `tools/fixup-project-versions.sh` to re-update the composer.lock after reverting the changes in the plugin directory?

Other than that, I don't know. It seems to work for me locally.

Reply:
That's what I was missing, thanks. It still updated the version for me and I thought we wanted to avoid that, so I reverted that, too.

Reply:
Sorry I wasn't clear. Yeah, it's mainly this one line in composer.json we wanted to leave out. 🙂